doc/modules/model_evaluation.rst

```diff
@@ -2005,6 +2005,19 @@ then the explained variance is estimated as follow:
 
 The best possible score is 1.0, lower values are worse.
 
+Note: when the prediction residuals have zero mean (perfectly unbiased
+model), the Explained Variance score and the :ref:`r2_score` are identical.
+
+In the particular case where the true target is constant, the Explained
+Variance score is not finite: it is either ``NaN`` (perfect predictions) or
+``-Inf`` (imperfect predictions). Such non-finite scores may prevent correct
```

Comment:
```suggestion
``-Inf`` (worst predictions). Such non-finite scores may prevent correct
```

Reply:
I disagree on this: even quasi-perfect predictions will lead to -Inf when y_true is constant.

Reply:
You are right. I misread.